### ISSUE TITLE
Markdown: using \dot{var} in html tables leads to markdown not being parsed correctly for html output

### ISSUE THREAD

**Issue by AndyHeather** [doxygen/doxygen#6806]
When writing html tables in markdown files, including the latex `\f$ \dot{var} \f$` causes the markdown text following the table to be rendered incorrectly. 

For example, this is OK:
```
<table border="0">
<tr>
    <td>\f$ A_a \f$</td>
    <td>=</td>
    <td>description 1</td>
</tr>
<tr>
    <td>\f$ B_b \f$</td>
    <td>=</td>
    <td>description 2</td>
</tr>
<tr>
    <td>\f$ C_c \f$</td>
    <td>=</td>
    <td>description 3</td>
</tr>
<tr>
    <td>\f$ D_d \f$</td>
    <td>=</td>
    <td>description 4</td>
</tr>
</table>


# Section 1 {#sec-1}

AAA

# Section 2 {#sec-2}

BBB

# Section 3 {#sec-3}

CCC


```
But this is not:
```
<table border="0">
<tr>
    <td>\f$ \dot{A_a} \f$</td>
    <td>=</td>
    <td>description 1</td>
</tr>
<tr>
    <td>\f$ B_b \f$</td>
    <td>=</td>
    <td>description 2</td>
</tr>
<tr>
    <td>\f$ C_c \f$</td>
    <td>=</td>
    <td>description 3</td>
</tr>
<tr>
    <td>\f$ D_d \f$</td>
    <td>=</td>
    <td>description 4</td>
</tr>
</table>


# Section 1 {#sec-1}

AAA

# Section 2 {#sec-2}

BBB

# Section 3 {#sec-3}

CCC

```

Using the `\f$ \dot{var} \f$` for inline math in the markdown file also works without problems.

Small test available here:  [TEST.tar.gz](https://github.com/doxygen/doxygen/files/2812191/TEST.tar.gz)

**Comment by albert-github** [comment]
I had a quick go at the example (TEST.tar.gz) and here I see that the rendering is not correct, but as far as I can see the incorrect rendering is caused by: `` `\dot{var}` `` replacing this by `\c \\dot{var}` solves this issue.

Did I overlook something.

**Comment by AndyHeather** [comment]
I'm trying to render this using math, i.e. `\f$ \dot{var} \f$` - all's ok for inline text, e.g.

`here is some markdown text with a dot over the var \f$ \dot{var} \f$`

- but when adding the same math text to a table cell, formatting after the table goes wrong (?)

**Comment by albert-github** [comment]
I see in the file TEST2.md there are on 2 places the `\f$ \dot{var} \f$`  construct and once the `` `\dot{var}` `` construct. In my test it looks like the construct in the backticks causes the problems. Replacing this solves the problem.

I also tried the example given in the question (see attachment) and this went OK.
(From the Doxyfile I see you use version 1.8.16 so the mater version, so I also used the current master version. When did you update last?).

My test: [mytest.tar.gz](https://github.com/doxygen/doxygen/files/2812427/mytest.tar.gz)
